### DISCUSSION TITLE
Including community suggested icons that don't make it into the icon set

### DISCUSSION THREAD

**Discussion by jguddas** [lucide-icons/lucide#1501] [Ideas]
There are a lot of icons that simply are not making it into the icon set, but it's still a shame when they get lost.

I would love if we somehow still have them for users who want to use them, be it as just as the SVG or a code snippet like this:

```tsx
const PooWithEyes = createLucideIcon('PooWithEyes', [
  ["path", { d: "M16.6 9A4.08 4.08 0 0 0 14 2a2 2 0 0 1-2 2h-2a2.95 2.95 0 0 0-2.2 5" }],
  ["circle", { cx: 8, cy: 13, r: 4 }],
  ["path", { d: "M8 13h.01" }],
  ["circle", { cx: 16, cy: 13, r: 4 }],
  ["path", { d: "M16 13h.01" }],
  ["path", { d: "M5.4 16H5a3 3 0 1 0 0 6h14a3 3 0 1 0 0-6h-.4" }]
]);
```

Just brainstorming, there could be an "unreleased icons" section on the website and people can vote on them to get them added.

---

> Seems a shame to waste them…
@danielbayley – https://github.com/lucide-icons/lucide/pull/1208#issuecomment-1676397570

**Comment by ericfennis** [comment]
See my comment in https://github.com/lucide-icons/lucide/issues/1607
